uw-frame-components/portal/misc/partials/circle-button.html

```diff
@@ -1,8 +1,9 @@
-<a ng-href="{{ cbDisabled ? '#' : href}}" target="{{cbDisabled ? '' : target}}">
-  <div class="btn btn-primary circle-button text-right" ng-class="{ 'btn-disabled' : cbDisabled}"> 
-    <i class="fa {{faIcon}} fa-2x"></i>
-  </div>
-</a>
+<md-button ng-href="{{ cbDisabled ? '#' : href}}"
+           target="{{cbDisabled ? '' : target}}"
+           class="md-fab md-mini md-primary"
```

Comment:
I think that's a good call. Normal (flat) icon buttons should suffice. I'll play around with it.